staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -359,6 +366,10 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 		{Version: version.MustParse("1.29"), Default: true, PreRelease: featuregate.GA, LockToDefault: true},
 	},
 
+	RemoteRequestHeaderUID: {
```

Comment:
also register this into `pkg/features/versioned_kube_features.go` like other generic feature gates